sqlite/src/comdb2lua.c

```diff
@@ -177,11 +175,12 @@ void comdb2CreateTrigger(Parse *parse, int dynamic, int seq, Token *proc,
 		return;
 	}
 
-	if (comdb2LocateSP(parse, spname) != 0) {
+	if (trigger_type != AUDIT_TRIGGER && comdb2LocateSP(parse, spname) != 0) {
```

Comment:
What happens when a SP with this name already exists? Is it ok to ignore it?

Reply:
It overwrites it. That's the functionality when you create a procedure

Reply:
Add a line to the docs explaining this behavior.